src/workerd/io/actor-cache.c++

```diff
@@ -173,12 +173,12 @@ kj::Maybe<kj::Promise<void>> ActorCache::getBackpressure() {
 
   // At one point, we tried applying backpressure if the total cache size was greater than
   // `softLimit`. This turned out to be a bad idea. If the cache is over the limit due to dirty
-  // entries waiting to be flushed, then `dirtyKeySoftLimit` will actually kick in first (since
-  // it's typically 64 keys, which equates to at most 2MB of data). So if the cache is over the
```

Comment:
FWIW this comment was quite out of date. We raised the max value size from 32 KiB to 128 KiB a long time ago, so 64 keys actually equated to a max of 8 MiB of data.

Which means that using 2 MiB as the new default could hypothetically cause performance degradation if someone was writing lots of max-length keys.

Is there a good reason not to just make this 8 MiB? I suppose that it could make for longer stalls for people writing smaller keys, since flushing a batch with more than 128 keys will take longer than flushing a batch with 64 or fewer keys. But at the same time this doesn't change when we start to flush, it only changes how soon we block additional requests from running in the isolate.

Maybe 4 MiB would be the happiest middle ground? 25% of the soft limit seems reasonable. But I'd prefer either 4 or 8 over 2. And 8 still seems fine unless I'm missing something (to be fair, I haven't thought this through as exhaustively as perhaps I should).